src/index.js

```diff
@@ -37,7 +37,31 @@ const useLocalStorage = (key, {updateFrequency}) => {
     }
   }, [])
 
-  return [value, writeToLocalStorage]
+  return [value, write]
 }
 
-export default useLocalStorage
+const useLocalStorageNoSync = key => {
+  const [value, setValue] = useState()
+
+  const read = () => {
+    const oldValue = value
+    const newValue = localStorage.getItem(key)
+    if (newValue !== oldValue) {
+      setValue(newValue)
+    }
```

Comment:
Shouldn't this return `newValue`?

Reply:
Returns `newValue` in 5e75fee394bee600d0207f1ec14506d8ff7b4b39